### ISSUE TITLE
make install completely user space

### ISSUE THREAD

**Issue by glyg** [CellModels/tyssue#3]
For now we need root privileges just to install the doc in `/usr/local`, that should change

**Comment by glyg** [comment]
happens around here:

https://github.com/CellModels/tyssue/blob/master/configure.ac#L310

Just need to grab the python package prefix and use this as default

**Comment by hadim** [comment]
NNNooooooo why not CMake ?????

You're going to regret it very soon :-)

**Comment by glyg** [comment]
I more or less copy/pasted Tiago's build system. Linking correctly with boost is kind of hellish (at least for me). Also he didn't need to change his configure files in the last 4 years although the code base changed a lot...

Why will I regret this? 

Now feel free for a CMake PR, I'll be more than happy to merge it :-)

**Comment by glyg** [comment]
And +1 beer for a `conda` recipe   ;-)

**Comment by hadim** [comment]
Ahahah !

Well it's more because autotools are old and CMake is new... I have no real technical arguments in favor of CMake, it's more a feeling :-D

For the conda recipe it's very easy (I've made one recently https://github.com/hadim/custom-conda-recipes) once you have local install working. See http://conda.pydata.org/docs/build_tutorials/pkgs2.html.

I used to master CMake some time ago (trying to convert Micro-Manager to CMake), so I'll see if I can help with an easy PR.
